src/Hyphen.OpenFeature.Provider.Tests/HyphenEvaluationContextTests.cs

```diff
@@ -0,0 +1,133 @@
+using OpenFeature.Model;
+using System.Net;
+
+namespace Hyphen.OpenFeature.Provider.Tests;
+
+public class HyphenEvaluationContextTests
+{
+    [Fact]
```

Comment:
FWIW this would be a good time to use a theory since all you are doing is changing the data input.

Reply:
@osbornm I did it, could you take a look?